cmd/libs/go2idl/conversion-gen/generators/conversion.go

```diff
@@ -57,9 +57,20 @@ func DefaultNameSystem() string {
 	return "public"
 }
 
+var fallbackPackages = []string{
+	"k8s.io/kubernetes/pkg/api/unversioned",
+	"k8s.io/kubernetes/pkg/apis/extensions",
+}
+
 func getInternalTypeFor(context *generator.Context, t *types.Type) (*types.Type, bool) {
 	internalPackage := filepath.Dir(t.Name.Package)
 	if !context.Universe.Package(internalPackage).Has(t.Name.Name) {
+		for _, fallbackPackage := range fallbackPackages {
```

Comment:
unversioned makes sense as a fallback package (I'd call it a "search order" though).

extensions-- hm, I guess I was thinking you'd look at what the various packages register but that might be hard. I'll think about this.